programmers/K번째수.js

```diff
@@ -0,0 +1,13 @@
+function solution(array, commands) {
+  let answer = [];
```

Comment:
```suggestion
  const answer = [];
```
이 배열 자체가 재 할당 되는 일이 없기 때문에 const로 선언해주세요